src/platforms/common/performance/custom-instrumentation.mdx

```diff
@@ -48,3 +48,9 @@ To instrument certain regions of your code, you can create transactions to captu
 <PlatformContent includePath="performance/distributed-tracing" />
 
 </PlatformSection>
+
+<PlatformSection supported={["dotnet"]}>
```

Comment:
ah, I see you added this already.

Reply:
looks duplicated though, could be as simple as

`<PlatformSection supported={["dotnet", "java"]}>`

above

`<PlatformContent includePath="performance/distributed-tracing" />`

so we don't need to duplicate this line

Reply:
I changed it, please take a look